package.json

```diff
@@ -45,10 +45,8 @@
     "sinon": "^7.2.3"
   },
   "dependencies": {
-    "aws-sdk": "^2.395.0",
-    "bluebird": "^3.5.3",
-    "request": "^2.88.0",
-    "request-promise": "^4.2.2",
+    "aws-sdk": "^2.407.0",
```

Comment:
Should `aws-sdk` be a peer dependency? Will reduce size even more

Reply:
up to you - I'm ok either way, dedupe should work (we can even lower the version caret)